l10n_it_fatturapa_out_triple_discount/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Questo modulo consente di generare correttamente la fattura elettronica (XML) quando il modulo account_invoice_triple_discount è installato.
+
+**English**
+
+The module allows to use correctly generate the Electronic Invoice (XML) when the module account_invoice_triple_discount is installed.
```

Comment:
@tafaRU Mi suona strana la frase in inglese

Reply:
A me no! Tra l'altro era già così nel modulo originale :)